src/System.Net.Primitives/src/System.Net.Primitives.csproj

```diff
@@ -11,12 +11,10 @@
     <ProjectGuid>{8772BC91-7B55-49B9-94FA-4B1BE5BEAB55}</ProjectGuid>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <EnableWinRT Condition="'$(TargetGroup)' == 'netcore50'">true</EnableWinRT>
-    <IsPartialFacadeAssembly Condition="'$(TargetGroup)' == 'net46'">true</IsPartialFacadeAssembly>
```

Comment:
I just submitted a partial fix, I'll submit a more complete fix in a few moments...